mui/base-ui
# [core] Enable stylelint on CSS files

I noticed we didn't lint the .css files after my IDE reported issues opening them (using https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint). It seems that we should start to lint those. There are too many rules to fix for me to make sense to do it, but I went with the simplest, quick win path:

- lint those path
- test if it works by fixing a few errors
- use the current number of warnings as the current limit accepted, so the problem doesn't grow

I think next we could:

- iterate on the rule to see the ones that make sense
- fix them all
- remove the warning default severity and max error count
- do the same on https://github.com/mui/material-ui
